ncm-dirperm/src/main/perl/dirperm.pm

```diff
@@ -31,6 +35,24 @@ sub Configure($$@) {
     # Define paths for convenience.
     my $base = "/software/components/dirperm";
 
+    my $filesystems = "/system/filesystems";
+    if ($config->elementExists($filesystems)) {
```

Comment:
Unfortunately we do not have a recent CCM for $config->getTree(), but I did the other things.